src/App.jsx

```diff
@@ -14,70 +16,67 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selected, setSelected] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selected ? (`${selected} is selected`) : (
+          'No goods selected'
+        )}
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+        {selected && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => {
+              setSelected('');
+            }}
+          />
+        )}
+      </h1>
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+      <table className="table">
+        <tbody>
+          {goods.map((good) => {
+            const isGoodSelected = good === selected;
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+            return (
+              <tr
+                data-cy="Good"
+                key={good}
+                className={cn('',
+                  { 'has-background-success-light': isGoodSelected })}
+              >
+                <td>
+                  <button
+                    data-cy={isGoodSelected ? 'RemoveButton' : 'AddButton'}
+                    type="button"
+                    className={cn('button',
+                      { 'is-info': isGoodSelected })}
+                    onClick={() => {
+                      if (isGoodSelected) {
+                        setSelected('');
+                      } else {
+                        setSelected(good);
+                      }
+                    }}
```

Comment:
```suggestion
                    onClick={() => {
                        setSelected(isGoodSelected ? '' : good);
                    }}
```

Reply:
Firstly I wrote like this, but linter didn`t like that and on Q&A mentor told to use if else

Reply:
oh, now I understood why. Thnx, I'll fix